### PR TITLE
Improve default target

### PR DESCRIPTION
**Pull request by sthaha**
Default make target (all) now brings up a cluster, builds operator and
bundle images, provides instructions on how to run the operator.

Signed-off-by: Sunil Thaha <sthaha@redhat.com>

### CODE REVIEW

**Comment by sthaha** [comment]
@vprashar2929 🙏

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,8 @@
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+THIS_FILE := $(lastword $(MAKEFILE_LIST))
```
Comment (vprashar2929) [inline Makefile:6]: Didn't know we could include make files like this😅

File: Makefile [hunk 2]
```diff
@@ -35,7 +37,23 @@ IMG_BASE ?= quay.io/sustainable_computing_io
 OPERATOR_IMG ?= $(IMG_BASE)/kepler-operator:$(VERSION)
 
 .PHONY: all
-all: build
+all: ## default target - setup a k8s cluster with operator installed
```
Comment (vprashar2929) [inline Makefile:40]: cluster-up doesn't take into account the env variable that we set [here](https://github.com/sustainable-computing-io/kepler-operator/blob/464c5221d96bc765833371b8e7510429889791bf/Makefile#L80). It brings up the cluster with default values.

**Comment by vprashar2929** [comment]
Also, I was thinking shouldn't we exclude cluster-up from make all and let the user run independently according to their needs? Let make all be specific to building and pushing operator and bundle images at one go

File: Makefile (hunk 2 above)
Comment (sthaha) [reply Makefile:40]: is the default value fine? Note, this is for a "default" make target - all

**Review by vprashar2929 — APPROVED** [review]
LGTM

**Pull request merged**